source/administration/object-management.rst

```diff
@@ -220,6 +220,48 @@ Conversion to or from XML may be required for importing rules created on S3 or s
 
 See :ref:`minio-lifecycle-management` for more complete documentation.
 
+Target Bucket Considerations
+----------------------------
+
+MinIO does *not* require that the target bucket match object management or versioning configurations with the source bucket.
+The target bucket *may* have its own set of object management rules, if defined with care.
+
+Target buckets should *not* have their own rules for expiration or additional tiering.
+Expiration rules can result in removal of tiered data still in use by the source bucket.
+Tiering to an additional remote creates an additional network hop between the hot tier and it's data while also increasing operational complexity.
+
+You *may* configure object locking or versioning on the remote bucket.
+Use caution, as mismatched rules between the source and the target buckets run a risk of data corruption and loss.
```

Comment:
:thinking: well not mismatched because the source *can* have either, but the remote does *not* require it. Spitballing

```suggestion
Enabling versioning or object locking on the remote is redundant at best and may result in unexpected behavior in response to delete operations.

Reply:
Or

```suggestion
```

and go right into line 236

Reply:
I'll just cut that line entirely. 👍